src/main/java/org/cadixdev/mercury/mixin/MixinRemapperVisitor.java

```diff
@@ -364,4 +364,45 @@ private static MethodSignature convertSignature(final String name, final IMethod
         return new MethodSignature(name, new MethodDescriptor(parameters, convertType(binding.getReturnType())));
     }
 
+    private void remapPrivateMixinTarget(
+        final AST ast, final TypeDeclaration typeDeclaration, final ITypeBinding typeBinding
+    ) {
+        IAnnotationBinding[] annotations = typeBinding.getAnnotations();
+        List modifiers = typeDeclaration.modifiers();
+        for (Object modifier : modifiers) {
+            if (modifier instanceof NormalAnnotation) {
+                NormalAnnotation normalAnnotation = (NormalAnnotation) modifier;
+                for (Object raw : normalAnnotation.values()) {
+                    MemberValuePair pair = (MemberValuePair) raw;
+                    if ("targets".equals(pair.getName().getIdentifier())) {
+                        StringLiteral literal = ((StringLiteral) pair.getValue());
```

Comment:
This can be either a `StringLiteral` or an `ArrayInitializer` depending on if the Mixin is specified like `@Mixin(targets = "NetworkManager$2")` or `@Mixin(targets = {"NetworkManager$2"})`

Reply:
changed but haven't tested